### ISSUE TITLE
Paste fails.

### ISSUE THREAD

**Issue by jordwalke** [t9md/atom-vim-mode-plus#38]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.3.0-beta6
**System**: Mac OS X 10.10.3
**Thrown From**: 'atom-vim-mode-plus' package, vundefined

### Stack Trace

Uncaught TypeError: fn is not a function

```
At /Users/jwalke/.atom/packages/atom-vim-mode-plus/lib/operation-stack.coffee:41

TypeError: fn is not a function
  at PutAfter.Operator.flash (/Users/jwalke/.atom/packages/vim-mode-plus/lib/operator.coffee:66:7)
  at PutAfter.PutBefore.pasteLinewise (/Users/jwalke/.atom/packages/vim-mode-plus/lib/operator.coffee:492:6)
  at /Users/jwalke/.atom/packages/vim-mode-plus/lib/operator.coffee:6:61
  at /Users/jwalke/.atom/packages/vim-mode-plus/lib/operator.coffee:472:7
  at TextBuffer.module.exports.TextBuffer.transact (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:798:18)
  at TextEditor.module.exports.TextEditor.transact (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/text-editor.js:1202:26)
  at PutAfter.PutBefore.execute (/Users/jwalke/.atom/packages/vim-mode-plus/lib/operator.coffee:471:13)
  at OperationStack.process (/Users/jwalke/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:73:8)
  at OperationStack.run (/Users/jwalke/.atom/packages/vim-mode-plus/lib/operation-stack.coffee:23:8)
  at run (/Users/jwalke/.atom/packages/vim-mode-plus/lib/base.coffee:14:27)
  at atom-text-editor.<anonymous> (/Users/jwalke/.atom/packages/vim-mode-plus/lib/base.coffee:137:38)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
  at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom Beta.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
  at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/window-event-handler.js:99:36)
  at HTMLDocument.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)

```

### Commands

```
     -0:30.3.0 pane:show-next-item (atom-text-editor.editor.vim-mode-plus.normal-mode.is-focused)
     -0:30.3.0 blur (atom-text-editor.editor.vim-mode-plus.normal-mode.is-focused)
     -0:29.4.0 vim-mode-plus:insert-after (atom-text-editor.editor.vim-mode-plus.normal-mode.is-focused)
     -0:25.5.0 vim-mode-plus:activate-normal-mode (atom-text-editor.editor.vim-mode-plus.is-focused.insert-mode)
     -0:25.1.0 core:save (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:24.4.0 vim-mode-plus:yank (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:24.2.0 vim-mode-plus:put-after (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:23.3.0 vim-mode-plus:yank (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  3x -0:19.4.0 blur (atom-text-editor.editor.vim-mode-plus.is-focused.operator-pending-mode)
     -0:18.8.0 core:save (atom-text-editor.editor.vim-mode-plus.operator-pending-mode.is-focused)
     -0:17.8.0 vim-mode-plus:move-up (atom-text-editor.editor.vim-mode-plus.operator-pending-mode.is-focused)
     -0:17.7.0 vim-mode-plus:move-down (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:17.5.0 core:save (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:17.2.0 vim-mode-plus:yank (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
     -0:16 vim-mode-plus:put-after (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
  2x -0:14.2.0 blur (atom-text-editor.editor.vim-mode-plus.is-focused.normal-mode)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "disabledPackages": [
      "minimap-selection",
      "nuclide-toolbar",
      "nuclide-debugger-atom",
      "nuclide-debugger-hhvm",
      "nuclide-debugger-lldb",
      "nuclide-test-runner",
      "nuclide-remote-projects",
      "wrap-guide",
      "atom-html-preview",
      "whitespace"
    ],
    "followSymlinks": false
  }
}
```

### Installed Packages

```coffee
# User
language-ocaml, v1.1.2
language-swift, v0.4.0
markdown-preview-opener, v0.1.1
toggle-gutter, v0.2.2
vim-mode-plus, v0.6.0

# Dev
No dev packages
```

**Comment by jordwalke** [comment]
Edit: This occurs on the Atom stable package too!

**Comment by t9md** [comment]
Released as v0.6.1.
Thank you very much for catching this.
The issue happened when flashOnOperate was disabled(default enabled).

Specs still lacks for default config parameter changes, need improvement.
